agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -460,30 +511,34 @@ async def create_image(
     elif image_type == "zip" and template_uri is None:
         raise Exception("template_uri must be provided for type zip")
 
-    image = ImageDB(
-        deletable=deletable,
-        user=user,
-    )
+    async with db_engine.get_session() as session:
+        image = ImageDB(
+            deletable=deletable,
+            user_id=user.id,
+        )
 
-    if image_type == "zip":
-        image.type = "zip"
-        image.template_uri = template_uri
-    elif image_type == "image":
-        image.type = "image"
-        image.tags = tags
-        image.docker_id = docker_id
+        if image_type == "zip":
```

Comment:
I introduced TemplateType let's use it instead

Reply:
Where? I can't find [it](https://github.com/Agenta-AI/agenta/blob/postgres/agenta-backend/agenta_backend/services/db_manager.py#L468-L474).